python/py-poetry-plugin-export/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-poetry-plugin-export
+version             1.0.6
+revision            0
+
+checksums           rmd160  9f422361331a37bbbae6003c68ebdf21bef72555 \
+                    sha256  af870afceb38e583afa57bcfadfa5cd35ebd74e35aacadcb802bb3a073c13adb \
+                    size    26355
+
+categories-append   devel
+supported_archs     noarch
+license             MIT
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+homepage            https://github.com/python-poetry/poetry-plugin-export
+description         A plugin that allows the export of locked packages to various formats.
+long_description    This plugin provides the same features as the existing export command \
+                    of Poetry which it will eventually replace.
+
+python.versions     37 38 39 310
+python.pep517       yes
+python.pep517_backend
```

Comment:
Needs to be set to `poetry`.

Reply:
This causes a dependency cycle if you do an install, though, as that flag sets a build requirement of py-poetry-core on py-poetry-core.

Reply:
I doesn't look like poetry-core depends on anything except (sometimes) importlib-metadata?

Reply:
Python portgroup does it https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/python-1.0.tcl#L350

Reply:
Yes, a dependency will be added if you do this, but it won't result in a cycle because py-poetry-core does not depend on py-poetry-plugin-export. The build of the latter is failing in CI because it does need py-poetry-core.

Reply:
My bad, ignore all of this, I thought we were discussing py-poetry-core and not this package.